app/selenium_ui/jsm/pages/customer_selectors.py

```diff
@@ -49,15 +49,16 @@ class CustomerPortalsSelectors:
     browse_portals_button = (By.CSS_SELECTOR, 'button.cv-smart-portal-browse-portals')
     full_portals_list = (By.CSS_SELECTOR, 'ul.cv-smart-portal-all-portals-list')
     portal_from_list = (By.CSS_SELECTOR, '"ul.cv-smart-portal-all-portals-list>li>a>span"')
-
+    new_portal_from_list = (By.CSS_SELECTOR, '"portal-card-item"')
 
 class CustomerPortalSelectors:
     portal_title = (By.CSS_SELECTOR, '.cv-page-title-text')
     request_type = (By.CSS_SELECTOR, 'li>span.js-cv-request-type>a')
     create_request_button = (By.XPATH, "//button[contains(text(),'Create')]")
     summary_field = (By.ID, 'summary')
     description_field = (By.ID, 'description')
-
+    new_description_field = (By.CSS_SELECTOR, '.ProseMirror')
+    new_version = (By.XPATH, "//*[contains(text(), 'jira/service-management')]")
```

Comment:
What does that mean  `new_version` selector? Can we specify selector naming e.g. `new_portal_version`?